exchanges/kucoin/kucoin_wrapper.go

```diff
@@ -0,0 +1,1364 @@
+	return nil, common.ErrFunctionNotSupported
+}
+
+// WithdrawFiatFundsToInternationalBank returns a withdrawal ID when a withdrawal is
+// submitted
+func (ku *Kucoin) WithdrawFiatFundsToInternationalBank(ctx context.Context, withdrawRequest *withdraw.Request) (*withdraw.ExchangeResponse, error) {
+	return nil, common.ErrFunctionNotSupported
+}
+
+// GetActiveOrders retrieves any orders that are active/open
+func (ku *Kucoin) GetActiveOrders(ctx context.Context, getOrdersRequest *order.GetOrdersRequest) (order.FilteredOrders, error) {
+	err := ku.CurrencyPairs.IsAssetEnabled(getOrdersRequest.AssetType)
+	if err != nil {
+		return nil, err
+	}
+	if getOrdersRequest.Validate() != nil {
+		return nil, err
+	}
+	format, err := ku.GetPairFormat(getOrdersRequest.AssetType, true)
+	if err != nil {
+		return nil, err
+	}
+	pair := ""
+	orders := []order.Detail{}
+	switch getOrdersRequest.AssetType {
+	case asset.Futures:
+		if len(getOrdersRequest.Pairs) == 1 {
+			pair = format.Format(getOrdersRequest.Pairs[0])
+		}
+		futuresOrders, err := ku.GetFuturesOrders(ctx, "active", pair, getOrdersRequest.Side.Lower(), getOrdersRequest.Type.Lower(), getOrdersRequest.StartTime, getOrdersRequest.EndTime)
+		if err != nil {
+			return nil, err
+		}
+		for x := range futuresOrders.Items {
+			if !futuresOrders.Items[x].IsActive {
+				continue
+			}
+			dPair, err := currency.NewPairFromString(futuresOrders.Items[x].Symbol)
+			if err != nil {
+				return nil, err
+			}
+			if len(getOrdersRequest.Pairs) == 1 && !dPair.Equal(getOrdersRequest.Pairs[x]) {
+				continue
+			} else if len(getOrdersRequest.Pairs) > 1 {
+				found := false
+				for i := range getOrdersRequest.Pairs {
+					if !getOrdersRequest.Pairs[i].Equal(dPair) {
+						continue
+					}
+					found = true
+				}
+				if !found {
+					continue
+				}
+			}
+			side, err := order.StringToOrderSide(futuresOrders.Items[x].Side)
+			if err != nil {
+				return nil, err
+			}
+			oType, err := order.StringToOrderType(futuresOrders.Items[x].OrderType)
+			if err != nil {
+				return nil, err
+			}
+			orders = append(orders, order.Detail{
+				OrderID:         futuresOrders.Items[x].ID,
+				Amount:          futuresOrders.Items[x].Size,
+				RemainingAmount: futuresOrders.Items[x].Size - futuresOrders.Items[x].FilledSize,
+				ExecutedAmount:  futuresOrders.Items[x].FilledSize,
+				Exchange:        ku.Name,
+				Date:            futuresOrders.Items[x].CreatedAt.Time(),
+				LastUpdated:     futuresOrders.Items[x].UpdatedAt.Time(),
+				Price:           futuresOrders.Items[x].Price,
+				Side:            side,
+				Type:            oType,
+				Pair:            dPair,
+			})
+		}
+	case asset.Spot, asset.Margin:
+		if len(getOrdersRequest.Pairs) == 1 {
+			pair = format.Format(getOrdersRequest.Pairs[0])
+		}
+		spotOrders, err := ku.ListOrders(ctx, "active", pair, getOrdersRequest.Side.Lower(), getOrdersRequest.Type.Lower(), "", getOrdersRequest.StartTime, getOrdersRequest.EndTime)
```

Comment:
You are not handling the `ANY` order type. Order type is an optional parameter, leave it empty when ANY is present.